units/units_z3.cpp

```diff
@@ -16,7 +17,8 @@ static Expr mkIntConst(const std::string name, ExprFactory &efac) {
   return bind::intConst(mkTerm(name, efac));
 }
 
-static Expr mkBvConst(const std::string name, ExprFactory &efac, unsigned width) {
+static Expr mkBvConst(const std::string name, ExprFactory &efac,
```

Comment:
why the extra new line?

overall, good, please merge

Reply:
IDE auto formatted presumably using `.clang-format`.